src/app/loaders/datadog-tracer.ts

```diff
@@ -0,0 +1,12 @@
+import tracer from 'dd-trace'
+
+tracer.init()
+
+// setup express to not track middlewares as spans
+// see documentation: https://datadoghq.dev/dd-trace-js/interfaces/plugins.express.html
+tracer.use('express', {
+  enabled: true,
```

Comment:
minor nit that since this defaults to `true`, might not be necessary?
```suggestion
```

from https://datadoghq.dev/dd-trace-js/interfaces/plugins.express.html

Reply:
Confirmed that `enabled:true` wasn't needed, and removed it in #c1698c5